connector/connect/client/jvm/src/main/scala/org/apache/spark/sql/streaming/StreamingQuery.scala

```diff
@@ -159,6 +185,21 @@ class RemoteStreamingQuery(
     executeQueryCmd(_.setStatus(true)).getStatus.getIsActive
   }
 
+  override def awaitTermination(): Unit = {
+    val awaitTerminationCmd = StreamingQueryCommand.AwaitTerminationCommand
+      .newBuilder()
+      .build()
+    executeQueryCmd(_.setAwaitTermination(awaitTerminationCmd))
+  }
+
+  override def awaitTermination(timeoutMs: Long): Boolean = {
+    val awaitTerminationCmd = StreamingQueryCommand.AwaitTerminationCommand
```

Comment:
This can simpler:

       executeQueryCmd(_.getAwaitTerminationBuilder.setTimeoutMs(timeout)). getAwaitTermination.getTerminated